### DISCUSSION TITLE
Facebook support?

### DISCUSSION THREAD

**Discussion by eswiig** [AAndyProgram/SCrawler#127] [Ideas]
Love SCrawler, just surprised there was no support for Facebook. Has it been discussed before and rejected?
I tried to search, issues and discussions, but saw no mention of it.

Would love if SCrawler could pull images and videos from facebook pages primarily, groups and user profiles would be a bonus.

There are a few different solutions, but none that can download consistently from facebook pages with lots of images and videos, most of them struggle, time out or freeze.

**Comment by AAndyProgram** [comment]
Sorry, I [won't add](https://github.com/AAndyProgram/SCrawler/blob/main/CONTRIBUTING.md#sites-i-will-never-develop) Facebook.
